<launch>
    <rosparam file="$(find ROBOT_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find ROBOT_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find ROBOT_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <node name="ROBOT_hardware_interface" pkg="ROBOT_hardware_interface" type="ROBOT_hardware_interface_node" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="
            /ROBOT/controller/state
            /ROBOT/controller/position/link_01_name__link_02_name
            /ROBOT/controller/position/link_02_name__link_03_name
        "/>
</launch>
